### ISSUE TITLE
response with not well formed JSON

### ISSUE THREAD

**Issue by fgalan** [telefonicaid/fiware-orion#772]
Observed in 0.19.0.

Given the following request:

```
(curl localhost:1026/ngsi10/contextEntities/cie_test_1/attributes/test_1 -s -S --header 'Content-Type: application/json' --header 'Accept: application/json' -d @- ) <<EOF
{"value":"50"}
EOF
```
Orion responses:

```
"statusCode" : {
  "code" : "200",
  "reasonPhrase" : "OK"
}
```

which is not a valid JSON (`{` and `}`) are missing.

The problem is severe enough to be solved in a branch of release/0.18.1 first, then ported to develop.

The problem could affect to other operations. A quick review of similar responses in the code would be great.

**Comment by fgalan** [comment]
I have checked that not only happens with 0.19.0, but also with HEAD in release/0.18.1 branch (commit: 92c402be29f21126127ef7a105e609752092dd8c)

**Comment by fgalan** [comment]
Fixed by PR #775 (release/0.18.1) and #776 (develop)
